### PR TITLE
feat: Include nft-did-resolver

### PR DESCRIPTION
**Pull request by ukstv**
Warning, alpha quality.

### CODE REVIEW

**Review by oed — COMMENTED** [review]
Kind of think we should update the nft-did-resolver to the new format before we merge: https://github.com/ceramicnetwork/CIP/pull/107

File: packages/cli/src/ceramic-daemon.ts [hunk 1]
```diff
@@ -182,6 +183,15 @@ export class CeramicDaemon {
       resolver: {
         ...KeyDidResolver.getResolver(),
         ...ThreeIdResolver.getResolver(ceramic),
+        ...NftDidResolver.getResolver({
+          ceramic: ceramic,
+          subGraphUrls: {
+            'eip155:4': {
+              erc721: 'https://api.thegraph.com/subgraphs/name/sunguru98/erc721-rinkeby-subgraph',
+              erc1155: 'https://api.thegraph.com/subgraphs/name/sunguru98/erc1155-rinkeby-subgraph',
+            },
```
[the remark this answers is not in the document]
Comment (ukstv) [reply packages/cli/src/ceramic-daemon.ts:192]: Not yet. It is alpha anyway.

[the remark this answers is not in the document]
Comment (ukstv) [reply packages/cli/src/ceramic-daemon.ts:192]: Added mainnet.

**Review by oed — APPROVED** [review]
Cool, would be good to get a checkmark from spencer as well.

**Comment by ukstv** [comment]
Actually, I removed Rinkeby endpoints. Found while writing a retro-active spec, that this thing relies on block time, which uses a hardcoded mainnet subgraph. We'll change it later, but for now it is main net only.

**Comment by oed** [comment]
oh, ok! Yeah, block time subgraph should also be configurable :)

**Review by stbrody — APPROVED** [review]
make me a little nervous to be using eth mainnet while this is fairly untested and in alpha, but if it's a lot of work to use an eth testnet then I guess it's not a huge deal.

We could still test this on the ceramic clay testnet right?  Ceramic testnet talking to eth mainnet for the NFTs?

**Comment by oed** [comment]
Yeah @stbrody the nft eth network usage is completely independent from what Ceramic uses.

**Pull request merged**